concepts/05 Widgets/File Manager/30 Validation/10 Permissions.md

```diff
@@ -0,0 +1,122 @@
+Use the [permissions](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/permissions/) option to specify whether to allow or deny a user to manage files and folders in the **FileManager**.
+
+You can manage user access to the following actions:
+
+- [copy](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/permissions/#copy) - Copy files and folders.
```

Comment:
The actions look pretty self-explanatory. I'd suggest mentioning the actions in the beginning instead of listing them. For example:

> Use the [permissions](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/permissions/) option to specify what a user can do with files and folders: create, download, move, remove, rename, or upload.

Reply:
So?